website/content/docs/commands/init.mdx

```diff
@@ -53,16 +53,25 @@ packer {
 }
 ```
 
+HashiCorp does not officially verify third party Packer plugins, plugins not under the HashiCorp namespace `hashicorp/*`; as with all open source tools, please do your own due diligence when using a new tool.
+
+## Plugin Selection
+Plugin selection depends on the source and version constraints defined within the `required_plugins` block.
+For each of the required plugins Packer will query the source repository `azr/myawesomecloud` whose fully qualified address
+is `https://github.com/azr/packer-plugin-myawesomecloud` for a plugin matching the version constraints for the host Os.
```

Comment:
"for the host 0s"?  What's the 0s?  Operating System? Can we say this? I read it as 0 not O